### PR TITLE
Add textbox for tags fix.

### LINKED ISSUE #3194: Tagging input appearing without logging in
The tagging input is appearing on the sidebar even when you aren't logged in. [Here's an example](https://publiclab.org/notes/warren/08-02-2018/use-image-sequencer-for-ndvi-plant-analysis-with-a-modified-mini-sport-camera).
tagging-logged-out

This is the logic deciding whether the input appears.
https://github.com/publiclab/plots2/blob/116db0cbaa8df72ea6cdb5e5115bff9eb5d6ee03/app/views/tag/_tagging.html.erb#L43

I believe the parent :profile inequality comparison is what is causing this. This is always true and thus displaying the input outside of the case for the profile view, here.

https://github.com/publiclab/plots2/blob/116db0cbaa8df72ea6cdb5e5115bff9eb5d6ee03/app/views/users/profile.html.erb#L127

Even then, I don't see a use for this comparison, because the desired behavior still checks out (if you are logged in and this is your content OR you are an admin, you can add tags).

It seems like a simple fix, and might be a good candidate for first time contributions.

### PR DESCRIPTION
**Pull request by tyrantdavis**
Fixes #3194 

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [X] code is in uniquely-named feature branch and has no merge conflicts
* [X] PR is descriptively titled
* [X] Fixes #3194
* [X] ask `@publiclab/reviewers` for help, in a comment below

@publiclab/reviewers - Hi folks. Will some please review my PR? I am requesting a merge. 

Thank you!

### CODE REVIEW

**Comment by tyrantdavis** [comment]
@publiclab/reviewers My test failed miserably.  Can I please get some help with this? That will be much appreciated. 

Thanks!

File: app/views/tag/_tagging.html.erb [hunk 1]
```diff
@@ -40,7 +40,7 @@ $(".label").on("click", function(e){
 
 <% parent ||= nil %>
 
-<% if parent != :profile || (current_user && (current_user.id == @user.id || current_user.role == "admin")) %>
+<% current_user && (current_user.id == @user.id || current_user.role == "admin") %>
```
[the remark this answers is not in the document]
Comment (tyrantdavis) [reply app/views/tag/_tagging.html.erb]: @gauravano Ah. Thanks for the feedback.

**Comment by gauravano** [comment]
I guess something is missing in FTO. Debugging it. Thanks!

**Comment by tyrantdavis** [comment]
@gauravano Happy to help!

**Comment by jywarren** [comment]
Ah, is this all good now? Thanks everyone!

**Comment by gauravano** [comment]
Its ready @jywarren

**Pull request merged**